README.md

```diff
@@ -48,3 +48,44 @@ by looking for at least one `coding` from the following known COPD Condition cod
   - "code": "J44.9"
 - "system": "http://snomed.info/sct"
   - "code": "404684003"
+
+## How To Run
+
+As a flask application, ``carl`` exposes HTTP routes as well as a number of command line
+interface entry points.
+
+These instructions assume a docker-compose deployment.  Simply eliminate the leading `dc exec`
+portion of each command if deployed outside of a docker container.
+
+```alias dc='sudo docker-compose'```
```

Comment:
This might be better in a "tips and tricks"-type section. I also personally think documentation that includes command invocations should be as verbose as possible, eg using long-options instead of flags. Using an alias might be confusing if someone only skimmed the documentation :wink: